InventoryService/cmd/controllers/item_controller.go

```diff
@@ -0,0 +1,205 @@
+package controllers
+
+import (
+	"encoding/json"
+	"inventoryService/m/v2/cmd/models"
+	"inventoryService/m/v2/cmd/resources"
+	"inventoryService/m/v2/ent"
+	"inventoryService/m/v2/ent/ingridient"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type ItemController struct {
+	Client *ent.Client
+}
+
+type Response struct {
+	Msg  string      `json:"message"`
+	Data interface{} `json:"data"`
+}
+
+func ResponseJSON(c *gin.Context, httpCode int, msg string, data interface{}) {
+	c.JSON(httpCode, Response{
+		Msg:  msg,
+		Data: data,
+	})
+}
+
+// @BasePath /api/v1
+
+// GetIngridientsList godoc
+// @Summary 	Gets list of all ingridients
+// @Description Get complete list of all ingridients availible for user
```

Comment:
Otstupy pojebalo((((

My recommendation is not to mix spaces and tabs, in this case only spaces should be better